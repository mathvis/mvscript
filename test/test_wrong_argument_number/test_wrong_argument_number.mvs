func a(b: int, c: float, d: bool) {}
a(5, 2.0);
