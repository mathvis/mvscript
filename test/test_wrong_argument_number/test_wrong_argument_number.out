error: Expected 3 arguments but got 2
 --> test/test_wrong_argument_number/test_wrong_argument_number.mvs:2:1
  |
2 | a(5, 2.0);
  | ^
  |
hint: Consider adding or removing arguments, or checking the function signature.
